src/Javis.jl

```diff
@@ -317,6 +329,91 @@ function get_javis_frame(video, objects, frame)
         # if object is in global layer this changes the background settings
         update_background_settings!(background_settings, object)
     end
+end
+
+function get_layer_frame(video, layer, frame)
+    Drawing(layer.width, layer.height, :image)
+    objects = layer.children
+    render_objects(objects, video, frame)
+    if frame in get_frames(layer)
+        # call currently active actions and their transformations for each layer
+        actions = layer.actions
+        if !isempty(actions)
+            for action in actions
+                #actions for layers are handled in a different way
+                # we don't calculate the relative frames for that
+                # can find out a way to pre_compute frames for actions applie to layers
+                # this is not a final fix..
+                if frame in get_frames(action)
```

Comment:
why isn't it relative though? Shouldn't it be similar to actions on objects?

Reply:
Yeah I actually wanted to discuss this with you and @TheCedarPrince. I tried to put the layers in to the `preprocess_frames` method with some minor tweaks. But kept getting errors and `act!` on layers was skewed by a few frames. So I remove the relative frames concept for now. I have tested and this works well for almost all the cases. 

But I'll surely try to revert that once I have a clear idea of the `preprocess_frames` method.